pkg/gateway/model_build_targetgroup.go

```diff
@@ -179,6 +180,14 @@ func (t *targetGroupModelBuildTask) BuildTargetGroup(ctx context.Context) error
 		return err
 	}
 
+	glog.V(6).Infof("svc.Spec.IPFamilies: %v\n", svc.Spec.IPFamilies)
```

Comment:
please dont use %v and \n, be specific with logging, no need to dump entire struct into log, choose fields that reader cares about

Reply:
Will remove in next revision. Was using that for debugging purpose 😅